contrib/debian/control

```diff
@@ -0,0 +1,60 @@
+Source: fwupd
+Priority: optional
+Maintainer: Debian EFI <debian-efi@lists.debian.org>
+Uploaders: Steve McIntyre <93sam@debian.org>,
+           Matthias Klumpp <mak@debian.org>,
+           Mario Limonciello <superm1@gmail.com>
+Build-Depends: gnu-efi [amd64 arm64 armhf i386],
+               meson,
+               mingw-w64-tools [amd64 arm64 armhf i386],
+Standards-Version: 4.5.0
+Section: admin
+Homepage: https://github.com/fwupd/fwupd-efi
+Vcs-Git: https://salsa.debian.org/efi-team/fwupd-efi.git
+Vcs-Browser: https://salsa.debian.org/efi-team/fwupd-efi
+
+Package: fwupd-efi-unsigned
+Architecture: amd64 arm64 armhf i386
+Depends: ${misc:Depends},
+         ${shlibs:Depends},
+Recommends: fwupd-efi-signed
+Suggests: gir1.2-fwupd-2.0
+Provides: fwupdate,
+          fwupd-signed
```

Comment:
@steve-mcintyre  specifically I'm not sure about this - I'm guessing it shouldn't provide fwupd-signed with this split.